include/na/nasp/Solver.hpp

```diff
@@ -4,6 +4,7 @@
 
 #include <cstddef>
 #include <cstdint>
+#include <ir/QuantumComputation.hpp>
```

Comment:
```suggestion
#include "ir/QuantumComputation.hpp"
```
If you apply the suggestion directly, it will be reordered by clang-tidy.